ClearKeep/Scenes/Home/Models/CallViewModel.swift

```diff
@@ -0,0 +1,454 @@
+//
+//  CallViewModel.swift
+//  ClearKeep
+//
+//  Created by HOANDHTB on 19/07/2022.
+//
+
+import Foundation
+import WebRTC
+import ChatSecure
+import ChatSecure
+
+class CallViewModel: NSObject, ObservableObject {
+	@Published var localVideoView: RTCMTLEAGLVideoView?
+	@Published var remoteVideoView: RTCMTLEAGLVideoView?
+	@Published var remotesVideoView = [RTCMTLEAGLVideoView]()
+	@Published var cameraOn = true
+	@Published var cameraFront = false
+	@Published var microEnable = true
+	@Published var speakerEnable = false
+	@Published var callStatus: CallStatus = .calling
+	@Published var callGroup = false
+	@Published var timeCall = ""
+	@Published var remoteViewRenderSize: CGSize = CGSize.zero
+	@Published var isVideoRequesting = false
+	@Published var callType: CallType = .audio
+	@Published var remotesVideoViewConfig = [String : CustomVideoViewConfig]()
+	var remotesVideoViewDict = [String : RTCMTLEAGLVideoView]()
+	var backHandler: (() -> Void)? = nil
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/Home/Models/CallViewModel.swift#L29"></span>Initializing an optional variable with nil is redundant.<br />
<code>redundant_optional_initialization</code> <code>CallViewModel.swift:29</code></td>
    </tr>
  </tbody>
</table>